lms/envs/devstack.py

```diff
@@ -15,6 +15,7 @@
 SITE_NAME = 'localhost:8000'
 # By default don't use a worker, execute tasks as if they were local functions
 CELERY_ALWAYS_EAGER = True
+HTTPS = 'off'
```

Comment:
@stvstnfrd: Should I squash these HTTPS changes into one commit?  Or should I leave them as two commits - one for LMS and one for CMS?